index.js

```diff
@@ -54,12 +56,13 @@ class CircuitBreaker {
             this.breaker.apply(this.breaker, args)
```

Comment:
If I pass a `shouldRetry` with a check to short circuit if 404 (for example), then does that still count against the breaker?